terraform/modules/fake_server_resources/fake_server_resources.tf

```
@@ -224,6 +277,133 @@ resource "kubernetes_deployment" "integration-tester" {
   }
 }
 
+# Dead letter topic to which undeliverable intake batch messages are sent
+resource "google_pubsub_topic" "validate_dead_letter" {
+  for_each = var.ingestor_pairs
+  name     = "${var.environment}-${each.key}-validate-dead-letter"
+}
+
+locals {
+  # GCP PubSub creates a service account for each project used to move
+  # undeliverable messages from subscriptons to a dead letter topic
```

```suggestion
  # undeliverable messages from subscriptions to a dead letter topic
```
The typo also exists in the original module